### ISSUE TITLE
No orthometric height returned when transforming "ETRS89" to "RD New+NAP height"

### ISSUE THREAD

**Issue by TimoSkyGeo** [pyproj4/pyproj#438]
Dear reader, 

RD New + NAP height (epsg:7415) uses an orthometric height.
When converting ETRS89 (epsg:4258 or epsg:4937) to RD New + NAP height, the original ellipsoidal height is returned, not the desired orthometric height. To demonstrate this, first I can show an example using the preferred Transformer method:
```Python

import pyproj

point = (53.160753042, 4.824761912, 42.8614)

transformer = pyproj.Transformer.from_crs("EPSG:4258", "EPSG:7415")
result = transformer.transform(*point)
print(result)
## returned: (117380.12731068721, 575040.4054018782, 42.8614)
```

Alternatively, the pyproj.Proj method was used to provide locally stored files to aid in the height conversion, obtained from [this Dutch site](https://www.nsgi.nl/geodetische-infrastructuur/producten/programma-rdnaptrans):

```Python
point = (4.824761912, 53.160753042, 42.8614)   ## Note: lat and lon are flipped for this method

proj1 = pyproj.Proj(init='epsg:4258')
proj2 = pyproj.Proj(init='epsg:7415', nadgrids='./rdcorr2018.gsb', geoidgrids='./nlgeo2018.gtx')

result = pyproj.transform(proj1, proj2, *point)
print(result)
## returned: 117355.96248492599, 574917.5359652427, 42.8614)

```

The preferred result as given by cs2cs is:
```bash
 > cs2cs -r +init=epsg:4258 +to +init=rdnap:rdnap -f %.4f <<EOF 
 > 53.160753042 4.824761912 42.8614
 > EOF
117380.1203     575040.3398 1.0000
```

However, ```pyproj.Proj(init='rdnap:rdnap')``` is not accepted by pyproj, resulting in the following error:
```bash
pyproj.exceptions.CRSError: Invalid projection: +init=rdnap:rdnap +type=crs: 
(Internal Proj Error: proj_create: cannot expand +init=rdnap:rdnap +type=crs)
```

Therefore, to compare with the pyproj result displayed above, epsg:7415 is used:
```bash
 > cs2cs -r +init=epsg:4258 +to +init=epsg:7415 -f %.4f <<EOF 
 > 53.160753042 4.824761912 42.8614
 > EOF
117380.1314     575040.4061 1.3523
```
This ~1m result is desired over the ~42m result returned by pyproj.
Is there a way to have pyproj return this desired orthometric height that I may have overlooked?

Thank you in advance.

Kind regards,
Timo

System info:
Machine: Linux-4.4.0-142-generic-x86_64-with-Ubuntu-16.04-xenial
Python: 3.5.2
PROJ: 6.1.1
PyProj: 2.3.1
Cython: None

**Comment by TimoSkyGeo** [comment]
Thank you snowman2 for the feedback. In your case +init=rdnap:rdnap does indeed not work, for I use the rdnap Proj package [proj-rdnap](https://packages.debian.org/sid/proj-rdnap). I failed to include this in the original post, for which I offer my apologies. 

It turns out that pyproj was unable to access the install directory of proj-rdnap. Through pyproj.datadir.append_data_dir(), I was able to get the height conversion working. Now `pyproj.Proj(init='rdnap:rdnap)` is accepted. Consider this ticket resolved.
